### PR TITLE
dependabot's tracking for 'pip' is failing

### PR DESCRIPTION
**Pull request by delaramamiri**
## Description

This PR will remove the dependency for pip module. Dependebot relies on the dependebot.yml file to check for dependencies and their updates. However, in ebpf-for-windows, we don't have dependencies with python packages. In order to verify that, I used `pip freeze` to check for the directories and packages related to python code. However, this was empty. 

## Testing

The yml file is modified. Now, if we take these steps, we see that pip dependency is removed. 

- Enable dependabot for the forked repo via 'github->Security->Dependabot'. (This option is available only on forked repos).

- Select the 'Dependabot' tab @ 'github->Insights->Dependency Graph'.

- (OPTIONAL) - Trigger a dependabot scan for pip (by clicking 'last checked xxx -> 'Check for updates' link)

issue related #1492

### CODE REVIEW

File: .github/workflows/ossar-scan.yml [hunk 1]
```diff
@@ -59,6 +59,6 @@ jobs:
       id: ossar
 
     - name: Upload results to Security tab
-      uses: github/codeql-action/upload-sarif@ec3cf9c605b848da5f1e41e8452719eb1ccfb9a6
```
Comment (Alan-Jowett) [inline .github/workflows/ossar-scan.yml:62]: Why was this changed? Same question for the other changes in this PR?

**Review by Alan-Jowett — CHANGES_REQUESTED** [review]
Can you please isolate the one change in dependabot.yml and exclude the others? It's not clear why you are changing the version of ebpf-verifier being used.

File: .github/workflows/ossar-scan.yml (hunk 1 above)
Comment (delaramamiri) [reply .github/workflows/ossar-scan.yml:62]: These are the changes from dependabot automatic PRs. Once they are merged, these changes will not show. 
Automatic PRs that I merged from dependabot can be found [here:](https://github.com/delaramamiri/ebpf-for-windows/pulls?q=is%3Apr+is%3Aclosed)

File: .github/workflows/scorecards-analysis.yml [hunk 2]
```diff
@@ -35,7 +35,7 @@ jobs:
           persist-credentials: false
 
       - name: "Run analysis"
-        uses: ossf/scorecard-action@b614d455ee90608b5e36e3299cd50d457eb37d5f # Don't update this until they fix PR support
+        uses: ossf/scorecard-action@99c53751e09b9529366343771cc321ec74e9bd3d # Don't update this until they fix PR support
```
Comment (dthaler) [inline .github/workflows/scorecards-analysis.yml:38]: This one should not be changed, as implied by the comment, or the scheduled build will break,

Comment (delaramamiri) [reply .github/workflows/scorecards-analysis.yml]: Reverted back to original

**Review by dthaler — COMMENTED** [review]
I think the ebpf-verifier change is updating to a bad version before Gianni reverted his changes.  Can you remove ebpf-verifier and catch2 from the PR?

**Comment by dthaler** [comment]
@delaramamiri we still need the dependabot.yml change, right?

**Comment by delaramamiri** [comment]
@dthaler , yes I was trying to revert back the external and ebpf-verifier files but the github did not allow me to push for the changes. Therefore, I had to close this out and open a new pull request.
